lib/rules/selector-max-specificity/README.md

```diff
@@ -109,17 +109,17 @@ The following patterns are _not_ considered problems:
 
 <!-- prettier-ignore -->
 ```css
-:global(.foo) .bar {}
+:host(.foo) .bar {}
 ```
 
 <!-- prettier-ignore -->
 ```css
-:local(.foo.bar) {}
+:host-context(.foo.bar) {}
 ```
 
 <!-- prettier-ignore -->
 ```css
-:local(.foo, :global(.bar).baz) {}
+:host-context(.foo, :host(.bar).baz) {}
```

Comment:
question
typo

Should it be `:host-context(.foo, :host(.bar), .baz) {}` instead?

Reply:
The config in the readme allows the specificity of two classes.

`:host-context(.foo, :host(.bar).baz) {}` after all the ignores is `.foo, .bar.baz`.

I think the intention was to illustrate how nesting selectors works when ignoring.
I'd rather keep it as is because it isn't wrong.

Reply:
Did I find a bug then?

https://stylelint.io/demo/#N4Igxg9gJgpiBcCAWEDOAXAtJAdumAHugBQB0AZhBADQAE8KGZARgIYBOAlKWwF6e1gAXwA6OMQzRZc+ImUo16jEjw6c6q-oNHic8ADYQwrffKp14Ac0NtTqrhoX3NA4SGohyAS30wAcqwAtnCIhEEADr6kYKio7uAQON6WCCDAYrS0IiDsAK6+qNnwghmZWSCoML5g6BDsmIGsBJio4TBgXt4d6ACeRbQA2tkADNQATNTD2XTA5V6WOHUwAMpV7bXshQiDRcrYibLo2QC6tELHpTpC8bjJAGJ1jeipAFaoifGw4XGI6TiZ2QwPV8+i8eH62X0rHwGGmpUBvRBYOkiWSLXQrBwUA4UAhIChMKOIDE1yEQA

Reply:
`:host` has its own specificity, if you also ignore that you get the expected outcome : https://stylelint.io/demo/#N4Igxg9gJgpiBcCAWEDOAXAtJAdumAHugBQB0AZhBADQAE8KGZARgIYBOAlKWwF6e1gAXwA6OMQzRZc+ImUo16jEjw6c6q-oNHickjNgh5CKhXX0q2XDQtXtNA4SGohyASwA2MAHKsAtnCIhP4ADl6kYKiozuBG7gDmCCDAYrS0IiDsAK5eqBnwgqlp6SCoMF5g6BDsmH6sBJioITBgbu6t6ACe+bQA2hkADNQATNQDGXTAJW7xONUwAMrlLVXseQh9+cqGxkQTJRYZALq0QkdFOkIxuAkAYtV16EkAVqhGMbAh0YgpOGkZGE6Xg8bjwPQyHlY+AwEyKAK6wNB0jiM0a6FYOCgHCg4JAkOh6AyYiuQiAA